.gitlab-ci.yml

```diff
@@ -27,6 +28,7 @@ Test:
   except:
   - dev
   - master
+  - feature/custom-downloads-july
```

Comment:
probably should delete before merging

Reply:
In fact, we should change this to:
```
- /^feature\/.*$/
```
so that branches that start with `feature` do not start the Test job (they will start `Test_N_Build:feature` instead)